tutorials/katacoda/thanos/2-lts/intro.md

```diff
@@ -1,7 +1,10 @@
-# In progress
+[Thanos](thanos.io) is a set of components that can be composed into a highly available metric system with unlimited storage capacity. It can be added seamlessly on top of existing Prometheus deployments.
 
-🚧 This tutorial is in progress. 🚧
+Thanos works in cloud native environments like Kubernetes as well as traditional ones. This course uses docker containers with pre-built docker images.
```

Comment:
Let's get rid of these 2 sentences, it's a step 2 so people already know what Thanos is.

Reply:
I added this step to specify that we are using `docker` here !!

Reply:
then change this to This course uses docker containers with pre-built docker images.